OpenRA.Game/GameRules/Ruleset.cs

```diff
@@ -67,19 +67,6 @@ public class Ruleset
 
 			foreach (var weapon in Weapons)
 			{
-				var projectileLoaded = weapon.Value.Projectile as IRulesetLoaded<WeaponInfo>;
```

Comment:
Uhm.... I don't think you wanted to remove this particular block.

Reply:
I did, look at the further part, it was an old duplicate